### PR TITLE
VZ-4489: Updated ES Log Data for Dashboard Enhancement. 

### PR DESCRIPTION
**Pull request by Abhijithang**
# Description

- Added more keywords for ES logs. 
- Updated Jenkins file with a new ENV. 

Fixes VZ-4489

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: pkg/test/framework/metrics/ginkgo_metrics.go [hunk 1]
```diff
@@ -84,7 +94,77 @@ func NewLogger(pkg string, ind string) (*zap.SugaredLogger, error) {
 	}
 
 	suiteUUID := uuid.NewUUID()
-	return log.Sugar().With("suite_uuid", suiteUUID).With("package", pkg), nil
+	sugaredLogger := log.Sugar().With("suite_uuid", suiteUUID).With("package", pkg)
+	return configureLoggerWithJenkinsEnv(sugaredLogger), nil
+}
+
+func getKubernetesVersion() (string, error) {
```
Comment (stevshap) [inline pkg/test/framework/metrics/ginkgo_metrics.go:101]: Do we have a function that does this already?

Comment (stevshap) [reply pkg/test/framework/metrics/ginkgo_metrics.go:101]: Looks like no

**Review by stevshap — APPROVED** [review]

File: pkg/test/framework/metrics/ginkgo_metrics.go (hunk 1 above)
Comment (Abhijithang) [reply pkg/test/framework/metrics/ginkgo_metrics.go:101]: getKuberentesVersion? not sure, I couldn't find it.  but there has been a usage of determining the k8s version using discovery client.

File: Jenkinsfile [hunk 2]
```diff
@@ -56,6 +56,7 @@ pipeline {
     }
 
     environment {
+        TEST_ENV = "JENKINS"
```
Comment (markxnelson) [inline Jenkinsfile:59]: this is ok for now, to get this in, but you should be able to work this out by looking for well known env vars that jenkins sets - or for the absence of this being set to LRE ..

**Review by markxnelson — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (Abhijithang) [reply Jenkinsfile:59]: All the variables in Jenkins file are being set to TEST_ENV variable, I checked across all Jenkins files, so we will get those filters. Ex: kind Jenkins file sets TEST_ENV to "Kind", we would have "Jenkins", "Kind", "LRE" in the filter dropdown. Similarly any new additions to other Jenkin files will work.

**Pull request merged**